python/python-core/intro-to-modules/what-are-modules.md

```diff
@@ -7,11 +7,13 @@ links:
 practiceQuestion:
   formats:
     - fill-in-the-gap
-  context: relative
+    - type-in-the-gap
+  context: standalone
 revisionQuestion:
   formats:
     - fill-in-the-gap
-  context: relative
+    - type-in-the-gap
```

Comment:
there is no value in having them type 1, 2, or 3 imo. we should reserve type for when it actually brings value to the learner
```suggestion
```